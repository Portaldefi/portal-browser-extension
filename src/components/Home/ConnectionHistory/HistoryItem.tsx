import React from 'react';
import { Grid, Header } from 'semantic-ui-react';

interface IConnectionItem {
  url: string,
  timestamp: string
};

export default ({ url, timestamp }: IConnectionItem) => {
  return (
    <Grid className='p-16 py-9 bg-white'>
      <Grid.Column width='sixteen' textAlign='left'>
        <Header as='p' className='description'>{url}</Header>
        <Header as='p' className='comment'>{timestamp}</Header>
      </Grid.Column>
    </Grid>
  )
}