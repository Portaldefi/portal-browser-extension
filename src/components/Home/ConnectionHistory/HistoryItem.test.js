import HistoryItem from './HistoryItem';
import { create, act } from 'react-test-renderer';

describe('HistoryItemComponent', () => {
  it('should work', () => {
    let tree;
    act(() => {
      tree = create(
        <HistoryItem />
      );
    });

    expect(tree).toMatchSnapshot();
  });
  afterAll(() => jest.resetModules());
});