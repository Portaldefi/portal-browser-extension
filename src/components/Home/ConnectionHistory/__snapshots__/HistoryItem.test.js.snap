// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HistoryItemComponent should work 1`] = `
<div
  className="ui grid p-16 py-9 bg-white"
>
  <div
    className="left aligned sixteen wide column"
  >
    <p
      className="ui header description"
    />
    <p
      className="ui header comment"
    />
  </div>
</div>
`;
